views/pages/login.ejs

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Pages shouldn't be full `<html>` pages, they should just have the content that's specific to an individual page. You should move the boilerplate into your `views/template.ejs`